src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -116,18 +116,18 @@
           </v-icon>
           {{ metadataButtonText }}
         </v-btn>
+        <!-- TODO for now only admins can publish -->
```

Comment:
```suggestion
```
I think this is just an artifact of a merge.

Reply:
https://github.com/dandi/dandiarchive/pull/796/commits/d4194bfef2bf4e5a8df4192ae89d8d99842567cf